by-language/csharp-npgsql/README.rst

```diff
@@ -37,34 +37,34 @@ To invoke a CrateDB instance for evaluation purposes, run::
 
 Invoke example program::
 
-    dotnet run --framework=net5.0
+    dotnet run --framework=net6.0
```

Comment:
here and below, maybe use `8.0`, the latest? but no strong opinion.

Reply:
Sure, why not. I've updated all those references to say `net8.0` instead.

Reply:
Now, we need to look into making Npgsql 8 work again.

- GH-169
- https://github.com/crate/crate-qa/pull/289